src/core_ct/core.py

```diff
@@ -52,4 +52,70 @@ def slice(self, axis, loc):
             case 2:
                 return self.pixel_array[:,:,loc]
             case _:
-                raise Exception("axis must be a value between 0 and 2 (inclusive)")
\ No newline at end of file
+                raise Exception("axis must be a value between 0 and 2 (inclusive)")
+
+    def trim(self, axis, loc_start, loc_end=None):
+        """Get a three-dimensional slice of the core scan trimming off the outside specified amount on the specified
+            axis.
+
+        Parameters:
+            axis -- integer either 0,1,2 specifying which dimension to collapse:
+                    0 corresponds to x-axis
+                    1 corresponds to y-axis
+                    2 corresponds to z-axis
+            loc_start -- integer value along the axis specifying the amount to trim off the smaller interval.
+            loc_end -- integer value along the axis specifying the amount to trim off the larger interval.
```

Comment:
Maybe say beginning and end

Reply:
Also add a note that it will be symmetric by default